src/thematic/0_generic/sfp-001_vervanging-terrest_nl/07_stappenplan.Rmd

```diff
@@ -0,0 +1,117 @@
+
+knitr::include_graphics("media/vervangingsprocedure_geenvervangpunt.png")
+
+```
+
+#### Wat als delen van de habitatkaartpolygoon niet toegankelijk zijn of niet beoordeeld kunnen worden?
+
+*Wanneer wordt bij BWK-kartering iets als ontoegankelijk beschouwd?*
+
+Wanneer delen van de habitatkaartpolygoon **permanent (of voor een lange tijd) ontoegankelijk** zijn (bv. omdat er een hek aanwezig is), mogen de vervangpunten die hierin vallen genegeerd worden en kan de vervangprocedure gewoon verder gezet worden. De ontoegankelijke delen vallen immers weg uit het steekproefkader en komen sowieso niet in aanmerking voor bemonstering.
+
+Wanneer delen van de habitatkaartpolygoon tijdelijk ontoegankelijk zijn (bv. door een uitzonderlijke overstroming) of niet beoordeeld kunnen worden (bv. er is recent gemaaid) zal in sommige gevallen de vervangingsprocedure niet kunnen doorgaan. We volgen hiervoor volgende regels:
+
++ de **vervanginsprocedure kan doorgaan** als een geschikt vervangpunt wordt gevonden met een rangnummer lager dan het laagste rangnummer in de tijdelijk ontoegankelijke of niet te beoordelen zone (bv. als in Figuur \@ref(fig:fig2) vervangpunten 6 en 8 tijdelijk ontoegankelijk zijn kan er wel een bemonstering in het geschikte vervangpunt 3 gebeuren);
+
++ de **vervanginsprocedure kan niet doorgaan** als het eerste geschikte vervangpunt een hoger rangnummer heeft dan het laagste rangnummer in de tijdelijk ontoegankelijke of niet te beoordelen zone (bv. als in Figuur \@ref(fig:fig2) vervangpunten 2 en 4 tijdelijk ontoegankelijk zijn kan het zijn dat vervangpunt 2 het gepaste vervangpunt is en niet vervangpunt 3).
+
+In het laatste geval wordt de vervangprocedure op een ander moment uitgevoerd.
+
+#### Wat als er meerdere vervangpunten gezocht moeten worden?
+
+Het komt voor dat er meerdere steekproefeenheden in een polygoon gelegen zijn en dat er meerdere vervangpunten geselecteerd moeten worden. Je gaat dan gewoon verder met [werkwijze 1](#werkwijze1) of [werkwijze 2](#werkwijze1) tot je een volgend geschikt vervangpunt vindt.
+
+Het kan zijn dat de veldwerkplanning voorschrijft dat de steekproefeenheden binnen een polygoon in verschillende jaren bemonsterd moeten worden. Dan kan je toch best tijdens je eerste bezoek alle steekproefeenheden evalueren en indien nodig de geschikte vervangingspunten selecteren. De eigenlijke bemonstering doe je dan in de jaren zoals de de veldwerkplanning het voorschrijft. Het voordeel is dan dat je de vervangingsprocedure maar eenmaal moet doorlopen en dat je tijdens een tweede bezoek rechtstreeks naar het geschikte punt kunt navigeren.    
+
+
+## Registratie en bewaring van resultaten
+
+Voor alle beoordeelde (potentiële) vervangpunten wordt minstens de aan- of afwezigheid van het doelhabitat genoteerd. Bij voorkeur wordt bij afwezigheid van het doelhabitat, het geobserveerde habitattype of rbb genoteerd.
```

Comment:
Ook de verschillende elementen van niet-toegankelijkheid / niet-beoordeelbaarheid, van de polygoon of delen ervan, is te noteren. Dit ook ivm todo's die hier uit kunnen volgen (bv. moet hernomen worden op meer geschikt tijdstip --> te noteren is dan wanneer).

Reply:
OK. Toegevoegd.